ops/ansible/roles/postmaster_deploy/tasks/main.yml

```diff
@@ -0,0 +1,55 @@
+---
+
+- name: Install main list of packages
+  apt: pkg={{item}} state=installed update_cache=true
+  become: yes
+  with_items:
+    - git
+    - python
+    - python-pip
+    - python-dev
+    - python-virtualenv
+    - libldap2-dev
+    - libssl-dev
+    - libsasl2-dev
+    - libffi-dev
+    - apache2
+    - libapache2-mod-wsgi
+    - libmysqlclient-dev
+  when: provision_type == "dev"
+
+- name: Create virtualenv directory
+  file: path=/opt/postmaster/env state=directory owner={{remote_user}} group={{remote_user}} mode=0775 recurse=yes
+  become: yes
+
+- name: Create logs directory
+  file: path=/opt/postmaster/logs state=directory owner={{remote_user}} group=www-data mode=0775 recurse=yes
+  become: yes
+
+- pip: requirements=/opt/postmaster/git/requirements.txt virtualenv=/opt/postmaster/env
+
+- include: mysql.yml
+  when: provision_type == "dev"
+
+- name: Enable wsgi module for apache2
+  apache2_module: state=present name=wsgi
+
+- name: Copy apache2 vhost config over
+  become: yes
+  copy: src=apache2/postmaster.conf dest=/etc/apache2/sites-available/postmaster.conf mode=0644
+
+- name: Disable default vhost
```

Comment:
http://blog.froehlichundfrei.de/2016/01/25/lets-migrate-to-omnibus-gitlab.html
https://github.com/chef/omnibus

The idea was to mimic chef's omnibus tool in ansible. The omnibus allows for a full stack installation of your software. I believe it should be done a fresh vanilla server. The gitlab docs recommend not using the omnibus package for installing on a server with other services. 

Maybe we can have it install to its own vhost? I'm not sure what's best for the user. Owncloud looks to make it a `/owncloud` alias, correct?